message_loop.h pulls in a lot of other headers (glib/x11/etc...) that these
files don't care about, so simplify that

--- a/synchronization/waitable_event_posix.cc
+++ b/synchronization/waitable_event_posix.cc
@@ -2,11 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include <algorithm>
+#include <vector>
+
 #include "base/synchronization/waitable_event.h"
 
 #include "base/synchronization/condition_variable.h"
 #include "base/synchronization/lock.h"
-#include "base/message_loop.h"
+#include "base/logging.h"
 
 // -----------------------------------------------------------------------------
 // A WaitableEvent on POSIX is implemented as a wait-list. Currently we don't
--- a/tracked_objects.cc
+++ b/tracked_objects.cc
@@ -5,9 +5,9 @@
 #include "base/tracked_objects.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "base/format_macros.h"
-#include "base/message_loop.h"
 #include "base/profiler/alternate_timer.h"
 #include "base/stringprintf.h"
 #include "base/third_party/valgrind/memcheck.h"
